packages/govuk-frontend-review/src/views/full-page-examples/search/index.njk

```diff
@@ -184,10 +184,15 @@ notes: >-
 
       {{ govukPagination({
         next: {
-          text: 'Next page',
-          href: '#',
-          labelText: '2 of 2939'
-        }
+          href: '#'
```

Comment:
We're slowly switching our Nunjucks to double quotes everywhere

Can we fix these at the same time? Thanks @querkmachine